query-cloudtrail.sh

```diff
@@ -2,13 +2,24 @@
 
 set -euo pipefail
 
-JQ_TRANSFORM=".Events[] | [{EventId: .EventId, EventName: .EventName, EventTime: .EventTime, Username: .Username}]"
+JQ_QUERY="
```

Comment:
I've never tried multiline jq like this, I'm assuming you tested it and it works as intended?

Reply:
yeah, works a charm